### PR TITLE
add 0.00001 to denominator when computing loss

### PR DESCRIPTION
**Pull request by FlyingQianMM**
* add 0.00001 to denominator when computing loss
* add weight when computing mutli-branch softmax loss
* add shape jugement before resize label when computing bce loss and dice loss

### CODE REVIEW

File: pdseg/loss.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def softmax_with_loss(logit,
         weighted_label_one_hot.stop_gradient = True
 
     loss = loss * ignore_mask
-    avg_loss = fluid.layers.mean(loss) / fluid.layers.mean(ignore_mask)
+    avg_loss = fluid.layers.mean(loss) / (fluid.layers.mean(ignore_mask) + 0.00001)
```
Comment (ZeyuChen) [inline pdseg/loss.py]: Use a global epsilon variable to store it, don't use hard code.

**Review by ZeyuChen — CHANGES_REQUESTED** [review]
用一个eps常量来存储1e-5,

File: pdseg/loss.py (hunk 1 above)
Comment (FlyingQianMM) [reply pdseg/loss.py]: Done. Thanks!

Comment (FlyingQianMM) [reply pdseg/loss.py]: Done. Thanks!

**Review by ZeyuChen — APPROVED** [review]
LGTM
